exercises/error-handling/common.go

```diff
@@ -2,6 +2,16 @@ package erratum
 
 import "io"
 
+// These are the support types and interface definitions used in the
+// implementation if your Use function. See the test suite file at
+// ./error_handling_test.go for information on the expected implementation.
```

Comment:
Personally I'd leave out the file name, the user should know where to find the test suite by this point, and it adds a level of fragility I think.